extensions/nato-phonetic-alphabet/src/spell.tsx

```diff
@@ -8,9 +8,8 @@ interface Arguments {
 const NEW_LINE = "\r\n\r\n";
 
 export default function Command(props: LaunchProps<{ arguments: Arguments }>) {
-  const term = props.arguments["Term"];
-
-  const words = term.split(" ");
+  const term = props.arguments["Term"] || props.fallbackText;
+  const words = (term || "").split(" ");
```

Comment:
**suggestion:** I'd do it so this so that Raycast product changes, like the introduction of fallback commands deliberately break, instead of silently do nothing.

```suggestion
  const term = props.arguments["Term"] || (props.fallbackText || "");
  const words = term.split(" ");
```